src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -106,11 +106,11 @@ private TestParameters WithScriptOptions(TestParameters parameters)
         }
 
         protected async Task TestDiagnosticMissingAsync(
-            string initialMarkup, TestParameters parameters = default)
+            string initialMarkup, TestParameters parameters = default, bool fixableDiagnosticsOnly = true)
```

Comment:
the idea of thees overloads is that all parameters go into the TestPrameters struct.

Reply:
also... why did we need to update all these test harnesses?  it's not clear to me the purpose.

Reply:
Ah, will move it into TestParameters, then we don't need to update any test helper.

Reply:
Thanks @CyrusNajmabadi, fixed. Now the changes are pretty minimal.